WordPress/wordpress-cloudflare-SSL-configuration.md

```diff
@@ -0,0 +1,33 @@
+{{{
+  "title": "WordPress Cloud Flare SSL Termination",
+  "date": "08-25-2015",
+  "author": "Earl Herman",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+### IMPORTANT NOTE

CenturyLink Cloud WordPress hosting is currently in a Limited Beta program with specific customers by invitation only and is not intended for production usage.

During the Limited Beta there is no production Service Level Agreement.

## Overview:
+
+There are multiple methods that could be used to terminate SSL for an existing CenturyLink Cloud WordPress site. This method covers using CloudFlare CDN's SSL Termination.
+
+### This article assumes the following:
+
+* Working knowledge of basic WordPress functionality
```

Comment:
I have updated the line in question.  Thank you for your help.

Is there anything I need to do to pull this forward?